src/styles/blocks/head-block/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+  }
+
+  &__item {
+    list-style: none;
+    display: flex;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__item a {
+    display: inline-block;
```

Comment:
Avoid using height property to align text vertically. It could lead to layout issues when text changes. Instead, use flexbox properties or padding.